data/abilities/command-and-control/T1001__data-obfuscation.yml

```diff
@@ -0,0 +1,18 @@
+- id: d5fc16c2-e8a6-42b6-9599-6dd046c7e7eb
+  name: Data Obfuscation
+  description: Ability for Data Obfuscation (ATT&CK T1001). Generated from Atomic
+    Red Team procedures; review and test safely before use.
+  tactic: command-and-control
+  technique:
+    attack_id: T1001
+    name: Data Obfuscation
+  platforms:
+    windows:
+      psh:
+        command: Get-Content "#{image_file}", "#{tar_file}" -Encoding byte -ReadCount
+          0 | Set-Content "#{new_image_file}" -Encoding byte
+    linux:
+      sh:
+        command: cat "#{script}" | base64 | xxd -p | sed 's/../& /g' | xargs -n1 |
+          xxd -r -p | cat "#{image}" - > "#{evil_image}"; strings "#{evil_image}"
+          | tail -n 1 | base64 -d | sh
```

Comment:
perhaps a more meaningful implementation would be to obfuscate the current agent executable on disk and save it as a new file rather than obfuscate or read some random file